loadtest/results/cross-vm-affinity-probe-2026-08-28/affinity_probe.py

```
@@ -0,0 +1,147 @@
+    for sid in sids:
+        key = "A" if sid % 2 == 0 else "B"
+        assignment[str(sid)] = key
+        ch = grpc.insecure_channel(BACKENDS[key]["grpc"])
+        stub = exercise_pb2_grpc.ExerciseServiceStub(ch)
+        req = exercise_pb2.AnalyzeRequest(
+            exercise_id=1, session_id=sid,
+            reference_source="cross-vm-affinity-probe", persona="BEGINNER", session_nonce="")
+        try:
+            resp = stub.StartAnalysis(req, metadata=md, timeout=15)
+            results.append((sid, key, bool(resp.success)))
+        except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)